data-processing/append_mortality_data.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/python
+
+
+class AppendMortalityData:
+
+    def __init__(self):
+        from pyspark.sql import SparkSession
+        import os
+
+        self.psql_user = os.environ['POSTGRESQL_USER']
+        self.psql_pw = os.environ['POSTGRESQL_PASSWORD']
+
+        self.spark = SparkSession \
+            .builder \
+            .appName("Write mortality data to DB") \
+            .getOrCreate()
+
+    def get_states(self, d):
+        # from pyspark.sql.types import StringType
+        # from pyspark.sql.functions import lpad
+
+        file = 's3a://data-engineer.club/aux/state_codes.csv'
+        states = d.spark.read.csv(file, inferSchema=True, header=True)
+        states = states.select(states.number.alias('state_num_code'),
+                               states.code.alias('state'))
+        states.show()
+        return states
+
+    def create_schema(self):
+
+        from pyspark.sql.types import StructType, StructField,\
+            IntegerType, StringType, DateType
+
+        schema = StructType([
+            StructField('state', IntegerType(), True),
+            #StructField('state_num', IntegerType(), True),
+            StructField('county_fips', IntegerType(), True),
+            StructField('date', DateType(), True),
+            # StructField('weekday', IntegerType(), True),
+            # StructField('manner', IntegerType(), True),
+            StructField('number', IntegerType(), True)
```

Comment:
I personally like to have a separate `Schemas` file somewhere, but it might depend on how many are used throughout the project.